website/src/pages/Compare/Compare.jsx

```diff
@@ -142,80 +138,28 @@ const Compare = () => {
-                <span>Total Allocs bytes</span>
-                <figure className="macrobench__Sidebar__line"></figure>
-                <span>Allocs bytes vtgate</span>
-                <figure className="macrobench__Sidebar__line"></figure>
-                <span>Allocs bytes vttablet</span>
-              </div>
               <div className="compare__carousel__container">
-                <Swiper
-                  direction={"vertical"}
-                  slidesPerView={1}
-                  spaceBetween={30}
-                  mousewheel={true}
-                  keyboard={{
-                    enabled: true,
-                  }}
-                  pagination={{
-                    clickable: true,
-                  }}
-                  modules={[Mousewheel, Pagination, Keyboard]}
-                  onSlideChange={handleSlideChange}
-                  initialSlide={currentSlideIndex}
-                  className="mySwiper"
-                >
-                  {dataMacrobench.map((macro, index) => {
-                    return (
-                      <SwiperSlide key={index}>
-                        <Macrobench
-                          data={macro}
-                          gitRefLeft={gitRefLeft.slice(0, 8)}
-                          gitRefRight={gitRefRight.slice(0, 8)}
-                          swiperSlide={SwiperSlide}
-                        />
-                        <MacrobenchMobile
-                          data={macro}
-                          gitRefLeft={gitRefLeft}
-                          gitRefRight={gitRefRight}
-                          swiperSlide={SwiperSlide}
-                          handleSlideChange={handleSlideChange}
-                          setCurrentSlideIndexMobile={
-                            setCurrentSlideIndexMobile
-                          }
-                          currentSlideIndexMobile={currentSlideIndexMobile}
-                        />
-                      </SwiperSlide>
-                    );
-                  })}
-                </Swiper>
+                {dataMacrobench.map((macro, index) => {
+                  return (
+                    <div key={index}>
+                      <Macrobench
+                        data={macro}
+                        gitRefLeft={gitRefLeft.slice(0, 8)}
+                        gitRefRight={gitRefRight.slice(0, 8)}
+                        swiperSlide={SwiperSlide}
+                      />
+                      <MacrobenchMobile
+                        data={macro}
+                        gitRefLeft={gitRefLeft}
+                        gitRefRight={gitRefRight}
+                        swiperSlide={SwiperSlide}
+                        handleSlideChange={handleSlideChange}
+                        setCurrentSlideIndexMobile={setCurrentSlideIndexMobile}
+                        currentSlideIndexMobile={currentSlideIndexMobile}
+                      />
```

Comment:
When changing the values that I want to compare, I do not see the loading effect on the `Macrobench(Mobile)` components.